debug_test.go

```diff
@@ -334,10 +334,10 @@ func TestPrefixedDebugger_Assert(t *testing.T) {
 	}
 
 	d := rootDebugger{fail: fail}.Push("FIZZ")
-	d.Assert(1 == 1, "fizz")
+	d.Assert(true, "fizz")
```

Comment:
You might be interested in adding [testifylint](https://golangci-lint.run/usage/linters/#testifylint) to golangci linters . In another PR .